<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_window</class>
 <widget class="QWidget" name="Qmain_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="Qhoriz_layout_main">
     <item>
      <layout class="QVBoxLayout" name="Qvert_layout_actions">
       <item>
        <widget class="QGroupBox" name="Qbox_general">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Actiuni Generale</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="Line" name="Qline_general">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_general_email">
            <property name="text">
             <string>Send E-mail</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_general_notif">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Notificari</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_general_lich">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Licitatii</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_general_rap">
            <property name="text">
             <string>Rapoarte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_general_act">
            <property name="text">
             <string>Actiune</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="Qspacer_general">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>162</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="Qbox_client">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Actiuni Client</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Line" name="Qline_client">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_client_add">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Adauga intrare</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_client_modif">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Modifica intrare</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_client_del">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Sterge intrare</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Qbutton_client_comp">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Vezi info comp.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="Qspacer_client">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>132</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="Qline_main">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Qvert_layout_main">
       <item>
        <widget class="QLabel" name="Qlabel_title">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CRM brought to you by The JoestarsÂ®</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Qhoriz_layout_search">
         <item>
          <widget class="QLineEdit" name="Qtext_search"/>
         </item>
         <item>
          <widget class="QPushButton" name="Qbutton_search">
           <property name="text">
            <string>Cauta</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="Qtable">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
